geotrek/tourism/filters.py

```diff
@@ -85,6 +85,12 @@ class TouristicEventFilterSet(ZoningFilterSet, StructureRelatedFilterSet):
         label=_("Provider"),
         choices=TouristicEvent.objects.provider_choices()
     )
+    place = ChoiceFilter(
+        field_name='place',
+        empty_label=_("Event place"),
+        label=_("Event place"),
+        choices=TouristicEventPlace.objects.all().values_list('pk', 'name')
```

Comment:
```suggestion
        choices=TouristicEventPlace.objects.values_list('pk', 'name')
```
all is redundant

Reply:
For this to work we need to add a custom queryset on `TouristicEventManager` (see its `provider_choices` method for `provider` filter)